tamr_unify_client/models/attribute/collection.py

```diff
@@ -0,0 +1,94 @@
+from tamr_unify_client.models.attribute.resource import Attribute
+from tamr_unify_client.models.base_collection import BaseCollection
+
+
+class AttributeCollection(BaseCollection):
+    """Collection of :class:`~tamr_unify_client.models.attribute.resource.Attribute` s.
+
+    :param client: Client for API call delegation.
+    :type client: :class:`~tamr_unify_client.Client`
+    :param data: JSON data representing this resource
```

Comment:
Given that this is specifically an attribute resource, I feel like saying just "resource" is a little vague. Instead, I would prefer "attribute resource". However, I know that is being a little nit-picky.